SU-HSDO/suhumsci
# SHS-5901: Update Component browser thumbnail for Testimonial

# READY FOR REVIEW

## Summary
- Update thumbnail for Testimonial components

## Need Review By (Date)
11/06

## Urgency
medium

## Steps to Test
1. Log in to the [Tugboat test site](https://pr1667-jmcohhfzutkd5teprc2nx3clbusbkaor.tugboatqa.com/user/login)
2. Create a new Flexible Page
3. Add a testimonial. Confirm that the thumbnail used in the "Add Component" dialog is the updated version

<img width="1187" alt="Screenshot 2024-10-28 at 9 26 38 AM" src="https://github.com/user-attachments/assets/b307f797-25bb-451f-a73a-66681f73dbeb">

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208650063350963
